engine/src/test/java/org/hibernate/validator/test/internal/constraintvalidators/hv/SafeHtmlValidatorTest.java

```diff
@@ -164,6 +164,20 @@ public void testAdditionalProtocols() {
 		assertNumberOfViolations( validator.validate( new Bar( "<div src='data:image/png;base64,100101' />" ) ), 1 );
 	}
 
+	@Test
+	@TestForIssue(jiraKey = "HV-817")
+	public void testPreserveRelativeLinks() throws Exception {
+		descriptor.setValue( "whitelistType", WhiteListType.RELAXED );
+		descriptor.setValue( "baseUri", "http://127.0.0.1" );
+
+		assertTrue( getSafeHtmlValidator().isValid( "<img src='/some/relative/url/image.png' />", null ) );
```

Comment:
While it was a surprise to me that I by default got validation failure for both relative images and inline data images (HV-1302), it's hard to know if someone depend on this behaviour to ensure their definition of _valid_ input. 

From my perspective I'd say it's probably both sensible and expected to support relative paths by default, but changing the default is scary.